addons/binding/org.openhab.binding.ihc/src/main/java/org/openhab/binding/ihc/internal/converters/OnOffTypeWSIntegerValueConverter.java

```diff
@@ -45,4 +52,16 @@ public WSIntegerValue convertFromOHType(@NonNull OnOffType from, @NonNull WSInte
                     + value.maximumValue + ")");
         }
     }
+
+    private int getCommandLevel(@NonNull WSIntegerValue value, @NonNull ConverterAdditionalInfo convertData,
+            Command command) throws ConversionException {
+        try {
+            if (convertData.getCommandLevels() != null) {
+                return (int) convertData.getCommandLevels().get(command);
+            }
+            return value.maximumValue;
+        } catch (Throwable e) {
```

Comment:
Do you need `Throwable` here? of can you limit to specific exception, or `RuntimeException`?